eng/Versions.props

```diff
@@ -7,8 +7,8 @@
   <PropertyGroup Label="Versioning">
     <RepositoryUrl>https://github.com/dotnet/dotnet-monitor</RepositoryUrl>
     <VersionPrefix>7.1.0</VersionPrefix>
```

Comment:
Didn't try it because I didn't think it would work for "going backward"; but it did seem to work: https://github.com/dotnet/dotnet-monitor/pull/3745/files

I'll close this PR